### PR TITLE
[CBRD-21524] fix warning: type qualifiers ignored on cast result type

### PR DESCRIPTION
**Pull request by valerinistor**
http://jira.cubrid.org/browse/CBRD-21524

Fix type qualifiers ignored on cast result type warning [-Wignored-qualifiers]

### CODE REVIEW

File: src/base/packer.cpp [hunk 1]
```diff
@@ -357,7 +357,7 @@ namespace cubpacking
   void
   packer::pack_int_vector (const std::vector<int> &array)
   {
-    const int count = (const int) array.size ();
+    const int count = (int) array.size ();
```
Comment (ribram) [inline src/base/packer.cpp]:
```suggestion
    const size_t count = array.size ();
```

File: src/executables/compactdb_cl.c [hunk 2]
```diff
@@ -831,7 +831,7 @@ compactdb_start (bool verbose_flag, bool delete_old_repr_flag, char *input_filen
       printf (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_COMPACTDB, COMPACTDB_MSG_PASS2));
     }
   status =
-    do_reclaim_addresses ((const OID ** const) class_oids, num_classes, &num_classes_fully_compacted, verbose_flag,
+    do_reclaim_addresses ((const OID **) class_oids, num_classes, &num_classes_fully_compacted, verbose_flag,
```
Comment (ribram) [inline src/executables/compactdb_cl.c]: Is cast necessary at all here?

Comment (valerinistor) [reply src/executables/compactdb_cl.c]: > Is cast necessary at all here?

Yes, otherwise: error: invalid conversion from ‘OID**’ to ‘const OID**’

Comment (ribram) [reply src/executables/compactdb_cl.c]: hm, I think the do_reclaim_addresses signature is actually incorrect. It probably should be `OID * const *class_oids`. What do you think?

Comment (valerinistor) [reply src/executables/compactdb_cl.c]: Yes, you're right

**Pull request merged**